src/negotiate.rs

```diff
@@ -374,6 +374,22 @@ impl Sspi for Negotiate {
             self.change_password(&mut yield_point, change_password).await
         }))
     }
+    
+    fn make_signature(&mut self,
+        flags: crate::SignatureFlags,
+        message: &mut [SecurityBuffer],
+        sequence_number: u32,
+    ) -> crate::Result<SecurityStatus> {
+        todo!()
```

Comment:
You can implement these methods for the negotiation module in the following way:
https://github.com/Devolutions/sspi-rs/blob/a8272f7497cff000bbed5a2a6f369b2d054adb72/src/negotiate.rs#L301-L313